app/lib/candidate_applications_helper.rb

```diff
@@ -0,0 +1,29 @@
+class CandidateApplicationsHelper
```

Comment:
I think this might fit better in a concerns module, potentially under services as it's used by a service, rather than a helper stored under `app/lib`.